cpgui_basics.py

```diff
@@ -265,7 +275,7 @@ def Update(self):
                 molmassstring='  N/A '
             if molmassstring=='' :
                 molmassstring='%6.2f'%(molmass*1000)
-            self.Ausgabetext+='Molar mass           :%s  \n'%molmassstring
+            self.Ausgabetext+=_('Molar mass           :%s  \n')%molmassstring
```

Comment:
Can you remove the .pot~ files from the PR please?

On Tue, Mar 24, 2015 at 2:19 PM, Reiner Mayers <notifications@github.com>
wrote:

> In cpgui_basics.py
> <https://github.com/CoolProp/GUI/pull/3#discussion_r27068541>:
>
> > @@ -265,7 +275,7 @@ def Update(self):
> >                  molmassstring='  N/A '
> >              if molmassstring=='' :
> >                  molmassstring='%6.2f'%(molmass*1000)
> > -            self.Ausgabetext+='Molar mass           :%s  \n'%molmassstring
> > +            self.Ausgabetext+=_('Molar mass           :%s  \n')%molmassstring
>
> I will change that on the next commit/push Ausgabetext => OutputText
> /Reiner Am 24.03.2015 um 17:46 schrieb Ian Bell:
> … <#14c4d70e24c54aab_>
>  In cpgui_basics.py <
> https://github.com/CoolProp/GUI/pull/3#discussion_r27047562>: > @@ -265,7
> +275,7 @@ def Update(self): > molmassstring=' N/A ' > if molmassstring==''
> : > molmassstring='%6.2f'%(molmass*1000) > - self.Ausgabetext+='Molar mass
> :%s \n'%molmassstring > + self.Ausgabetext+=_('Molar mass :%s
> \n')%molmassstring Please keep the code in the codebase in english — Reply
> to this email directly or view it on GitHub <
> https://github.com/CoolProp/GUI/pull/3/files#r27047562>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CoolProp/GUI/pull/3/files#r27068541>.
>